File: src/management/server/SmartSignalsManagementApi/Responses/ListResourcesResponse.cs
//-----------------------------------------------------------------------
// <copyright file="ListResourcesResponse.cs" company="Microsoft Corporation">
//        Copyright (c) Microsoft Corporation.  All rights reserved.
// </copyright>
//-----------------------------------------------------------------------

namespace Microsoft.Azure.Monitoring.SmartAlerts.Appliance.ManagementApi.Responses
{
    using System.Collections.Generic;
    using Microsoft.Azure.Monitoring.SmartAlerts.Appliance.ManagementApi.Models;
    using Newtonsoft.Json;

    /// <summary>
    /// This class represents the GET Management API operation for listing resources.
    /// </summary>
    public class ListResourcesResponse
    {
        /// <summary>
        /// Gets or sets the resources
        /// </summary>
        [JsonProperty("subscriptionResources")]
        public List<AzureSubscriptionResources> SubscriptionResources { get; set; }
    }
}

File: src/management/client/src/api/models/ListResourcesResponse.ts
// -----------------------------------------------------------------------
// <copyright file="ListResourcesResponse.ts" company="Microsoft Corporation">
//        Copyright (c) Microsoft Corporation.  All rights reserved.
// </copyright>
// -----------------------------------------------------------------------

import AzureSubscriptionResources from '../../models/AzureSubscriptionResources';

/**
 * Represents the response model for the GET /resource endpoint
 */
export default class ListResourcesResponse {
    /**
     * Gets or list of Azure resources
     */
    public subscriptionResources: ReadonlyArray<AzureSubscriptionResources>;
}